tdrs-backend/tdpservice/parsers/schema_defs/tribal_tanf/t5.py

```diff
@@ -25,13 +25,13 @@
             ]),
         ],
         postparsing_validators=[
+            category3.validate__FAM_AFF__SSN(),
```

Comment:
@lhuxraft @klinkoberstar @raftmsohani -- we need to decide if this should be deprecated in this ticket or #3562 . as it stands, we would have more tribal file changes to deprecate SSN-related validation.

Reply:
update: looks like this change isnt substantive. so we'll take care of the deprecation in #3562?

Reply:
@ADPennington I think we decided not to remove the check from Tribal validators, and then Kathryn to discuss and decide how/when we want to apply this change on Tribal, correct @klinkoberstar?